pkg/platform/abstract/project/external/leader/iguazio/client.go

```diff
@@ -36,6 +36,42 @@ func NewClient(parentLogger logger.Logger, platformConfiguration *platformconfig
 	return &newClient, nil
 }
 
+func (c *Client) Get(getProjectOptions *platform.GetProjectsOptions) ([]platform.Project, error) {
```

Comment:
It's a matter of style but I would differentiate between Get and List
their behavior is different, for example, on get I would expect not found to raise, on list empty list is totally fine

Reply:
I see what you are saying. Nuclio platform convention is using `Get` to get multiple records, then on the client (nuctl/dashboard) they perform the verification upon the CRUD itself.